src/js/rx/components/ErrorView.jsx

```diff
@@ -58,9 +58,10 @@ class ErrorView extends React.Component {
             status="warning"/>
         );
     }
+
     return (
       <div className="call-out rx-app-error rx-tab-explainer">
-        // needed due to call-out class css rule targeitng
+        // needed due to call-out class css rule targeting
```

Comment:
I think there was some selector that caused the first `p` element to have special styling, hence that class and the extra `<p></p>` at the top. If that class is only in FL now, we can probably remove it. Maybe it will no longer cause a visual issue but I haven't checked